server/utils/routes.js

```diff
@@ -6,4 +8,11 @@ export const routes = {
   newChart: '/charts/new',
   login: `/login`,
   signup: `/signup`,
+  logout: `/logout`,
+}
+
+export const v1Routes = {
+  adminRoutes: {
+    usersUpdatePermissions: `${v1Root}/admin/users/permissions/:uid`,
```

Comment:
let's talk about this route some more. What this endpoint actually does is update a user's attributes with whatever is in the request body. In the controller this looks like

```js
      await appDb
        .collection(collections.users)
        .updateOne({ uid }, { $set: req.body })
```

I think this is a really nice, extensible handling function but if we name it as updating permissions then we lose the ability to re-use it for other user updates. Why not rename it something generic so that we can use it for updating user permissions or any other user attributes?

Reply:
Makes sense
[8814b2b](https://github.com/AMP-SCZ/dpdash/pull/294/commits/8814b2b15da0a94568320a77593fb7c635f9ffb9)